EpsilonAuxBmsV2/Core/Src/Tasks/MemoryDebugTask.c

```diff
@@ -0,0 +1,28 @@
+#include "MemoryDebugTask.h"
+
+void memoryDebugTask(void* arg)
+{
+    uint32_t prevWakeTime = osKernelSysTick();
+
+    for (;;)
+    {
+        // Get the minimum
+        volatile size_t minimumHeapSize = xPortGetMinimumEverFreeHeapSize();
```

Comment:
We made it volatile because these variables aren't being used anywhere, so the compiler would probably optimize them out. Our plan is to print them out while using GDB.